### PR TITLE
chore: add cds theme to query string in demo app

### PR DESCRIPTION
**Pull request by kevinbuhmann**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [N/A] Tests for the changes have been added (for bug fixes / features)
- [N/A] Docs have been added / updated (for bug fixes / features)
- [N/A] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] Other... Please describe: Demo app changes

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by steve-haar — APPROVED** [review]

File: projects/demo/src/app/cds-theme-path-location-strategy.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+export class CdsThemePathLocationStrategy extends PathLocationStrategy {
+  constructor(
+    platformLocation: PlatformLocation,
+    @Optional() @Inject(APP_BASE_HREF) baseHref: string,
+    private readonly urlSerializer: UrlSerializer
+  ) {
+    super(platformLocation, baseHref);
+  }
+
+  prepareExternalUrl(internal: string): string {
```
Comment (ashleyryan) [inline projects/demo/src/app/cds-theme-path-location-strategy.ts:22]: Might be helpful to add a comment here

Comment (kevinbuhmann) [reply projects/demo/src/app/cds-theme-path-location-strategy.ts]: Comment added. Let me know if that doesn't make sense. I also added `override` keyword for clarity/maintainability.

https://github.com/vmware-clarity/ng-clarity/compare/6feead3954e6b2c8267a626bca77dd1f7b35304d..ff6910f69c625849ac15b051166cb8e4775ca9dd

Comment (ashleyryan) [reply projects/demo/src/app/cds-theme-path-location-strategy.ts]: LGTM, I suppose it's a very literal comment of what is happening, but I think it's helpful to have it called out that we want to keep the theme persisted in all of our URLs for convenience when sharing.

**Review by ashleyryan — APPROVED** [review]

File: projects/demo/src/app/cds-theme-path-location-strategy.ts (hunk 1 above)
Comment (kevinbuhmann) [reply projects/demo/src/app/cds-theme-path-location-strategy.ts]: Okay, I updated the commit. I moved the comment to the class as I feel that makes more sense. I think the `override` keyword would point the reader in the right direction for that method specifically.

Comment (ashleyryan) [reply projects/demo/src/app/cds-theme-path-location-strategy.ts]: Thank you! I was fine with the original version too, btw, but I appreciate the clarification.

**Pull request merged**